presto-product-tests/conf/presto/etc/catalog/hive.properties

```diff
@@ -7,7 +7,7 @@
 
 connector.name=hive-cdh5
 hive.metastore.uri=thrift://hadoop-master:9083
-hive.metastore.thrift.client.socks-proxy=hadoop-master:1080
+hive.metastore.thrift.client.socks-proxy=hadoop-master:1180
```

Comment:
Please mind my previous comment.
This change will be correct once I publish new docker image.

Please remove that line and change comment accordingly.
I will put the configuration back to `1180` when docker image with socks proxy listening on `1180` is published.